<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>profile</class>
 <widget class="QDialog" name="profile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#profile{

	background-color: qlineargradient(spread:pad, x1:0.280788, y1:0.0113636, x2:0.330039, y2:1, stop:0 rgba(161, 228, 231, 255), stop:1 rgba(255, 255, 255, 255));
}
*{
font-family:&quot;Times New Roman&quot;;
}

.QLabel{

background-color: rgb(85, 85, 255) ;
color: white; 
border-radius : 10px ;
padding: 5px ;



}

.QLineEdit{
border-radius: 10px ;

}</string>
  </property>
  <widget class="QLabel" name="icon_name">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>63</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="icon_dob">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Date of Birth</string>
   </property>
  </widget>
  <widget class="QLabel" name="icon_email">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>410</y>
     <width>63</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Email:</string>
   </property>
  </widget>
  <widget class="QLabel" name="icon_company">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>460</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Currenlty Associated Company:</string>
   </property>
  </widget>
  <widget class="QLabel" name="nam_label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>310</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="dob_label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>360</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="emaillabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>410</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="company_label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>460</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="edit_profie">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>540</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit profile</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_name">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>310</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="icon_passwordold">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>460</y>
     <width>111</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Old Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_pwold">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>460</y>
     <width>311</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="dob_dateEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>360</y>
     <width>181</width>
     <height>26</height>
    </rect>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_update">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>540</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>670</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="coverPhoto">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1251</width>
     <height>281</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#coverPhoto{
border-image: url(:/images/images/cover.jpg); 
border-radius: 10px ; 
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>660</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Sign Out</string>
   </property>
  </widget>
  <widget class="QLabel" name="icon_passwordnew">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>510</y>
     <width>111</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>New Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="newPW_lineEdit">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>510</y>
     <width>311</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="changePW_PushButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>540</y>
     <width>141</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Change Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="updatePW_pushbutton">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>560</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>update</string>
   </property>
  </widget>
  <widget class="QLabel" name="icon_plan">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>350</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Next Plans</string>
   </property>
  </widget>
  <widget class="QLabel" name="icon_departureDate">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>390</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Departure Date</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_plan">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>350</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_departDate">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>390</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="seePlans">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>540</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>See Plans</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_book">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>360</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Cancel Book</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
